### PR TITLE
Release note for 18.1.0

### PR DESCRIPTION
**Pull request by rajiesh**

### CODE REVIEW

File: source/partials/release_notes/_release-18-1-0.md.erb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      (Use <%= link_to('latest version', 'https://github.com/cnorthwood/gocd-tls-auth/releases') %> of TLS Client Certificates Login instead.)
+      </p></li>
+
+      <li><p><%= link_to('Strong Auth Plugin', 'https://github.com/danielsomerfield/go-strong-auth-plugin#go-strong-auth-plugin') %>
+      (Use the bundled <%= link_to('gocd-filebased-authentication-plugin', 'https://github.com/gocd/gocd-filebased-authentication-plugin') %> instead.)
+      </p></li>
+      <li><p><%= link_to('Guest Login', 'https://github.com/gocd-contrib/gocd_auth_plugin_guest_user') %></p></li>
+    </ul>
+
+* <%= link_to_commit '23eb465f0b366378c526c70642c3d0748ce47e2e', 'Delete cruft from the DB for jobs that are completed and also for new jobs as they complete', 2875%>
```
Comment (varshavaradarajan) [inline source/partials/release_notes/_release-18-1-0.md.erb]: Remove this? You have already stated

File: source/partials/release_notes/_release-18-1-0.md.erb [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    * Artifact plans that are of type \"file\" (Artifact plans of type \"unit\" are retained for shine)
+
+    * Artifact properties generators
+
+    * Agent resources associated with a job
+
+    * Elastic agent metatada
+
+
+<h4>Breaking Changes</h4>
```
Comment (varshavaradarajan) [inline source/partials/release_notes/_release-18-1-0.md.erb]: Plugin info API v2 is removed

Comment (varshavaradarajan) [inline source/partials/release_notes/_release-18-1-0.md.erb]: Add a docker images section? We have introduced alpine 3.7.

File: source/partials/release_notes/_release-18-1-0.md.erb (hunk 1 above)
Comment (jyotisingh) [reply source/partials/release_notes/_release-18-1-0.md.erb]: I think the line about `This improvement breaks jobs feed API. We will no longer render the environment variables, resources and build artifacts in the job xml api` needs to be added to the previous mention of this fix.

File: source/partials/release_notes/_release-18-1-0.md.erb [hunk 3]
```diff
@@ -4,22 +4,10 @@
...
-
-    * Artifact plans that are of type \"file\" (Artifact plans of type \"unit\" are retained for shine)
-
-    * Artifact properties generators
-
-    * Agent resources associated with a job
-
-    * Elastic agent metatada
-
+<h4>Docker Images</h4>
```
Comment (varshavaradarajan) [inline source/partials/release_notes/_release-18-1-0.md.erb]: Also the server image is on alpine 3.7 now

**Pull request merged**